account_payment_partner/models/account_invoice.py

```diff
@@ -20,11 +20,15 @@ class AccountInvoice(models.Model):
 
     @api.onchange('partner_id', 'company_id')
     def _onchange_partner_id(self):
+        if self.company_id:
+            company_id = self.company_id
```

Comment:
Change variable name (in other places too):
```suggestion
            company = self.company_id
```

Reply:
@jcoux I keep this variable name to be constant in the function, because of this : https://github.com/OCA/bank-payment/pull/654/files#diff-b7b06a5eb2ba2f2489eb2d427fc5023aR42
But I can change If you want

Reply:
I think it's better to change it.

Reply:
I agree with the suggested change: local variables and function arguments ending with `_id` should be reserved to variables containing ids, not record sets.